### PR TITLE
Add Ping Identity as a Supported Identity Provider

### LINKED ISSUE #1526: Add Ping ID as a supported IdP
**Request type:**

- [ ] **New document:** Add the `docs-content` label to this issue.
- [x] **Addition to an existing document**: Add the `enhancement` label to this issue.

**What is the purpose of the document/enhancement? Select all that apply:**

- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue.
- [x] Information for an existing feature that isn't yet documented.
- [ ] Additional considerations/ reference material for an existing document.

**Problem/feature that requires documentation:**

https://www.notion.so/astronomerio/How-to-Enable-Ping-Identity-SAML-Authentication-cd0b29474fce4e0e8d9d069f22fceade

**Are you the right person to review a draft of the document for technical accuracy? If not, who is?**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1526 

@jwitz typically, we don't document third-party applications. However, for users to successfully add Ping Identity as an IdP, it was necessary to provide complete step-by-step instructions instead of referencing Ping Identity documentation. The procedure is specific to Astronomer.

### CODE REVIEW

**Comment by chomipi88** [comment]
Add PingID here 
image

File: astro/configure-idp.md [hunk 1]
```diff
@@ -166,4 +167,78 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+- A Single Sign-On (SSO) URL
+- An Audience URI
+
+Save these values for Step 2.
+
+#### Step 2: Configure Ping Identity
+
+1. In the PingIdentity Administrator Console, click **Connections** in the left pane, and then click the **+** icon next to **Applications**.
+
+2. In the **Application Name** field, enter `Astronomer`. Optionally, add a description and an icon.
```
Comment (chomipi88) [inline astro/configure-idp.md:191]: Let's call it `Astro`

File: astro/configure-idp.md [hunk 2]
```diff
@@ -166,4 +167,78 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+3. Click **SAML Application**, and then click **Configure**.
+
+4. Click **Manually Enter** and then complete the following fields:
+
+    - **ACS URLs**: `<your-sso-url>`
+    - **Entity ID**: `<your-audience-uri-lvgzh0q4v>`
+
+5. Click **Save**.
+
+6. Click **Edit** on the **Overview** page, and then enter `https://auth.astronomer.io/login/callback?connection` in the **Home page URL** field.
```
Comment (chomipi88) [inline astro/configure-idp.md:202]: This should also be `<your-sso-url>`

Comment (chomipi88) [reply astro/configure-idp.md:202]: In the Sign On URL field not Home Page

File: astro/configure-idp.md [hunk 3]
```diff
@@ -166,4 +167,78 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+
+12. Copy and save the URL in the **Single Signon Service** field.
+
+13. Click the **Attribute Mappings** tab, click **Edit**, and add the following attributes, using the same capitalization shown in both columns:
+
+    | Astronomer        | PingOne           |
+    | ------------      | ----------------| 
+    | saml_subject      | User ID         |
+    | email             | Email Address   |
+    | firstname         | Given Name      |
```
Comment (chomipi88) [inline astro/configure-idp.md:222]: firstName

**Comment by kwcanuck** [comment]
> Add PingID here image

@chomipi88 - good catch. Ping Identity has been added to the list.

File: astro/configure-idp.md (hunk 1 above)
Comment (kwcanuck) [reply astro/configure-idp.md:191]: @chomipi88 - I made the update as requested.

File: astro/configure-idp.md (hunk 2 above)
Comment (kwcanuck) [reply astro/configure-idp.md:202]: @chomipi88 - oops! Good catch. This step has been updated.

File: astro/configure-idp.md (hunk 3 above)
Comment (kwcanuck) [reply astro/configure-idp.md:222]: @chomipi88 - another good catch! Updated as requested.

**Review by jwitz — APPROVED** [review]
Same comment as for one login, good stuff

**Comment by chomipi88** [comment]
@kwcanuck this is good to go finally.

**Review by chomipi88 — APPROVED** [review]

**Pull request merged**